src/components/Chart/Chart.tsx

```diff
@@ -168,3 +200,37 @@ function getRangeAreaEnd<XDomain>(ra: RangeArea<XDomain>, domain: AxisDomain): s
   if (Array.isArray(domain)) return domain.includes(ra.end as any) ? (ra.end as any) : domain[domain.length - 1]
   return Math.min(+ra.end, +domain.end)
 }
+
+function renderRangeAreaStroke<XDomain>(rangeAreas: RangeArea<XDomain>) {
+  return rangeAreas.strokeColor
+    ? [
+        <ReferenceLine
+          yAxisId='data'
+          stroke={rangeAreas.strokeColor}
+          x={typeof rangeAreas.init === 'string' ? rangeAreas.init : +rangeAreas.init}
+          position='start'
+        />,
+        <ReferenceLine
+          yAxisId='data'
+          stroke={rangeAreas.strokeColor}
+          x={typeof rangeAreas.end === 'string' ? rangeAreas.end : +rangeAreas.end}
+          position='start'
+        />,
+      ]
+    : []
+}
+
+function defMaskPattern() {
```

Comment:
extract this into a .svg file

Reply:
I couldn’t do it, so I changed to a specific TS file for the pattern